codingTest_kjy/flag에따라다른값반환하기.js

```diff
@@ -0,0 +1,7 @@
+function solution(a, b, flag) { 
+    var answer = 0;              
```

Comment:
var를 사용하면 안되는 이유는 너무 많이 올린 내용이라 밑에 링크 참조해주세요.
```suggestion
    let answer = 0;              
```
https://github.com/modern-agile-team/7term-assignment/pull/4#discussion_r1413403588